README.md

```diff
@@ -1,14 +1,11 @@
 # Arigato 
```

Comment:
Note that this is a README file. In the future, you'd want to create a JavaScript file with extension `.js`.